README.rst

```diff
@@ -14,6 +14,16 @@ This library includes various cross-cutting concerns related to APIs. API functi
 
 Some of these concerns include extensions of `Django REST Framework <http://www.django-rest-framework.org/>`_ (DRF), which is how the repository initially got its name.
 
+Publishing a Release
+--------------------
+
+After a PR merges, a new version of the package will automatically be released by Travis when the commit is tagged. Use::
+
+    git tag -a X.Y.Z -m "Releasing version X.Y.Z"
+    git push origin X.Y.Z
```

Comment:
**Notes for Reviewers**: Is this what people do?  I always used Github Releases, so not sure what belongs here. :)

Reply:
Probably not in scope here, but do we have a standard for `vX.Y.Z` vs. `X.Y.Z`? Some repos seem to use a mix. :-)

Reply:
> Maybe we could add a recommendation to the Semantic Versioning OEP?

That sounds about right!